docs/wiki/Client-Installation-and-Configuration.md

```diff
@@ -95,7 +95,7 @@ wallet.
 3. **Litecoin.** Either [Litecoin Core v0.21.x](https://litecoin.org/) or
    [Electrum-LTC v4.2.x](https://electrum-ltc.org/) are supported.
 4. **Dogecoin.** [Dogecoin Core v1.14.5+](https://dogecoin.com/).
-5. **ZCash.** [zcashd v5.1](https://z.cash/download/).
+5. **Zcash.** [zcashd v5.1](https://z.cash/download/).
```

Comment:
While here, please change the zcashd version to v5.4.2.  That is the minimum as of d1c45817f3d22db3d93a3dbc9cfb7715546ec2c1